shared/common/flake.nix

```diff
@@ -57,7 +57,9 @@ in
 
     services.telegraf.extraConfig.inputs.file =
       let
-        inputsWithDate = lib.filterAttrs (_: input: input ? lastModified) cfg.flake.inputs;
+        inputsWithDate = lib.filterAttrs (_: input: input ? lastModified) (
+          cfg.flake.inputs // { inherit (cfg) flake; }
```

Comment:
Doesn't this generate a rebuild on every commit? I would prefer not to having to rebuild all my machines everytime.

Reply:
But it's only relevant if srvos.flake is set and in that case you already get rebuilds on each commit, as the flake is included in the closure, no?

Or are you referring to telegraf restarts specificially?